### PR TITLE
PAYARA-1153 Enable to turn on Weld development mode and the Probe tool

### PR DESCRIPTION
**Pull request by jGauravGupta**
Available options to enable Weld Probe tool :

**web.xml :**

```
    <context-param>
        <param-name>org.jboss.weld.development</param-name>
        <param-value>true</param-value>
    </context-param>
```

**System Properties :**

```
-Dorg.jboss.weld.development=true
```

**Console :**

```
asadmin deploy --properties cdiDevModeEnabled=true sample.war
```

### CODE REVIEW

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
Hi @smillidge ,

Just for confirmation, administration console means here a UI (checkbox) option in deployment page.

**Comment by ggam** [comment]
Great to see this being worked on! This is a killer feature for development stage!

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by smillidge** [comment]
yeah a checkbox on the deployment page.

**Comment by smillidge** [comment]
jenkins test please

**Comment by mikecroft** [comment]
fixes #1001

File: appserver/admingui/core-l10n/src/main/resources/org/glassfish/admingui/core/Strings_es.properties [hunk 1]
```diff
@@ -913,6 +913,8 @@ deploy.deploymentOrder=Orden de Despliegue:
 deploy.deploymentOrderHelp=N\u00famero que determina el orden de carga de la aplicaci\u00f3n al iniciar el servidor. Los n\u00fameros inferiores se cargan en primer lugar. El valor por defecto es 100.
 deploy.deploymentTime=Tiempo de Despliegue:
 deploy.deploymentTimeHelp=El tiempo necesario para desplegar la aplicaci\u00f3n en milisegundos.
+deploy.cdiDevMode=Modo de desarrollo de CDI
+deploy.cdiDevModeHelp=Permite inspeccionar los componentes del CDI de la aplicaci\u00f3n en tiempo de ejecuci\u00f3n
```
Comment (ggam) [inline appserver/admingui/core-l10n/src/main/resources/org/glassfish/admingui/core/Strings_es.properties]: Better say "componentes *de* CDI" over "del CDI"

**Comment by smillidge** [comment]
@jGauravGupta can you resolve @ggam's comment

**Comment by smillidge** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

**Pull request merged**